### ISSUE TITLE
FP: oelint.vars.srcuridomains for SRC_URI override entries

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#313]
the following pattern gives a false positive oelint.vars.srcuridomains warning

```
SRC_URI:class-devupstream = "git://foo.bar;branch=master;protocol=https"
SRC_URI = "git://foo.baz;branch=master;protocol=https"
```
